src/traces/treemap/draw_ancestors.js

```diff
@@ -141,7 +141,9 @@ module.exports = function drawAncestors(gd, cd, entry, slices, opts) {
             s.attr('data-notex', 1);
         });
 
-        var font = Lib.ensureUniformFontSize(gd, helpers.determineTextFont(trace, pt, fullLayout.font, trace.pathdir));
+        var font = Lib.ensureUniformFontSize(gd, helpers.determineTextFont(trace, pt, fullLayout.font, {
+            onPathbar: true
```

Comment:
Can you explain why this commit made three existing mocks change (`treemap_level-depth`, `treemap_packings` and `treemap_textposition`) ?

Reply:
Good question. 
And a bad mistake here. 
There is no such a thing as `pathdir`!
It should have been written `pathbar`.